CHANGELOG.md

```diff
@@ -5,10 +5,19 @@
 ### Breaking Changes
 
 - Your generated code Assembly Definition file now needs to have `allow unsafe code` selected to compile. [#1255](https://github.com/spatialos/gdk-for-unity/pull/1255)
+- The `RedirectedProcess.WithArgs` API will now concatinate arguments, instead of replace the previously provided arguments. [#1260](https://github.com/spatialos/gdk-for-unity/pull/1260)
```

Comment:
```suggestion
- The `RedirectedProcess.WithArgs` API will now concatinate arguments, instead of replacing the previously provided arguments. [#1260](https://github.com/spatialos/gdk-for-unity/pull/1260)
```